src/main/java/org/kaazing/netx/ws/internal/WebSocketTransition.java

```diff
@@ -17,6 +17,8 @@
 
 public enum WebSocketTransition {
     RECEIVED_UPGRADE_RESPONSE,
+    RECEIVED_CONTINUATION_FRAME,
+    SEND_CONTINUATION_FRAME,
     RECEIVED_CLOSE_FRAME,
     SEND_CLOSE_FRAME,
     RECEIVED_PING_FRAME,
```

Comment:
My preference would be RECEIVE/SEND, assuming RECEIVE transition is what we do when a continuation frame is received. RECEIVING and RECEIVED sound like states rather than transitions.